docs/04-Captures/cli.md

```diff
@@ -1,203 +1,279 @@
 # Capture with Retina CLI
 
-Retina capture command allows the user to capture network traffic and metadata for the capture target, and then send the capture file to the location by Output Configuration.
+The Retina capture command allows users to capture network traffic and metadata for the capture target, and send the data to the location defined by output configuration.
 
-Note: captures can also be performed with a [Capture CRD](../05-Concepts/CRDs/Capture.md) after [installing Retina](../02-Installation/01-Setup.md) with capture support.
+> Note: captures can also be performed with a [Capture CRD](../05-Concepts/CRDs/Capture.md) after [installing Retina](../02-Installation/01-Setup.md) **with capture support**.
 
-## Retina capture create
+## Starting a Capture
 
-`kubectl retina capture create` creates a Capture with underlying Kubernetes jobs.
+`kubectl retina capture create [--flags]` creates a Capture with underlying Kubernetes jobs.
 
-### No wait(default true)
+## Flags
 
-Do not wait for the long-running capture job to finish.
+### [required] Name
 
-By default, Retina capture CLI will exit before the jobs are completed. With `--no-wait=false`, the CLI will wait until the jobs are completed and clean up the Kubernetes resources created.
+- Syntax: `--name <string>`
 
-#### Example
+A name for the Retina Capture.
 
-- do not wait for the long-running capture job to finish
+**Example:**
 
-`kubectl retina capture create --name capture-test --host-path /mnt/capture --node-selectors "kubernetes.io/os=linux" --no-wait=true`
+`kubectl retina capture create --name capture-test --host-path /mnt/capture --node-selectors "kubernetes.io/os=linux"`
 
-### Namespace(default "default")
+---
 
-Namespace to host capture job and the other k8s resources for a network capture. Please make sure the namespace exists.
+### [required] Capture Target
 
-#### Example
+The capture target indicates where the packet capture will be performed. There are three choices here, each of which is its own flag. They are described below.
```

Comment:
Suggestion
The target flag indicates...

This is assuming that target is a flag since it's under "Flags"

Reply:
The "Target" itself is not a flag but a concept which is required by one of three options described in that section.